cmd/witnessctl/witnessctl.go

```diff
@@ -24,12 +24,30 @@ import (
 	"os"
 )
 
+const warning = `
+████████████████████████████████████████████████████████████████████████████████
+
+                                **  WARNING  **
+
+Enabling NXP HABv4 secure boot is an irreversible action that permanently fuses
+verification keys hashes on the device.
```

Comment:
"keys hashes" is a bit odd - "key hashes"? or "hashes of verification keys" if you want it to be clear about pluralization.